docs/insights/api.md

```diff
@@ -0,0 +1,511 @@
+| Items | An array of the reports returned for the current page.  See the section [Get a specific Insights Report](#get-a-specific-insights-report) for details of the report structure |
+| Links | Links to other resources.  See REST API Links https://octopus.com/docs/octopus-rest-api#api-links |
+
+
+## Create an Insights report
+POST: api/{spaceId}/insights/reports
+Create a new Insights report.
+
+### Parameters
+| Name | Description |
+| ---- | ----------- |
+| spaceId | Id of the space | 
+
+### Request Body
+The request body is in JSON format.
+
+### Sample request
+```json
+{
+    "SpaceId": "Spaces-1",
+    "Name": "High Volume Report",
+    "Description": "High volume tenanted deployments report",
+    "ProjectGroupIds": ["ProjectGroups-1", "ProjectGroups-2"],
+    "ProjectIds": ["Projects-1", "Projects-2"],
+    "TenantIds": ["Tenants-1", "Tenants-2", "Tenants-3"],
+    "TenantTags": [
+      "Deployment Group/Group A",
+      "Deployment Group/Group B",
+      "Deployment Group/Group C",
+      "Location/Australia",
+      "Location/Asia",
+      "Location/Europe",
+      "Location/US"
+    ],
+    "TenantMode": "TenantedAndUntenanted",
+    "EnvironmentGroups": [
+      {
+        "Name": "Test",
+        "Environments": ["Environments-1", "Environments-2"]
+      },
+      {
+        "Name": "Internal",
+        "Environments": ["Environments-3", "Environments-4"]
+      },
+      {
+        "Name": "Production",
+        "Environments": ["Environments-5", "Environments-6"]
+      }
+    ],
+    "TimeZone": "UTC"
+}
+```
+
+### Properties
+| Property |  | Description |
+| -------- | -------- | ----------- |
+| SpaceId  | Required | The id of the space to contain the report | 
+| Name     | Required | The name of the report | 
+| Description | Optional | The description of the report | 
+| ProjectGroupIds | Optional | An array of the project group ids to be included in the report | 
+| ProjectIds | Optional | An array of the project ids to be included in the report | 
+| TenantIds  | Optional | An array of the tenant ids to be  included in the report | 
+| TenantTags | Optional | An array of the tenant tags  to be associated with this report | 
+| TenantMode | Optional | Indicates what types of deployments will be contributing to this report. May be TenantedAndUntenanted, Tenanted, or Untenanted | 
+| EnvironmentGroups | Optional | An array of the environment groups to be  included in this report.  See below for details | 
+| TimeZone | Optional | The timezone to be used in the report. Must be either a Windows or IANA (Tzdb) timezone |
+
+### EnvironmentGroups Properties
+| Property |  | Description |
+| -------- | -------- | ----------- |
+| Name | Required | The name of the environment group.  Must be unique per report |
+| Environments | Required | An array of the ids of the environments to include in the group |
+
+
+## Get a specific Insights report
+GET: api/{spaceId}/insights/reports/{reportId}
```

Comment:
I don't know if the leading slash should be added. I checked my API, and it included one
```suggestion
GET: `/api/{spaceId}/insights/reports/{reportId}`
```